core/src/callback.jl

```diff
@@ -496,6 +491,12 @@ function update_basin(integrator)::Nothing
         set_table_row!(table, row, i)
     end
 
+    for (i, id) in enumerate(basin.node_id)
+        update_vertical_flux!(basin, storage, i)
+    end
+
+    # Forget about vertical fluxes before basin update
+    copyto!(vertical_flux_prev, vertical_flux)
```

Comment:
To be most accurate, maybe an exception should be made for vertical fluxes which are not set by the `Basin \ time` table.

Reply:
Typically only some of the vertical fluxes will be in `Basin \ time`. Could you make an issue about this? I don't fully oversee the error introduced in this case.